src/inbox/mod_inbox.erl

```diff
@@ -143,6 +149,11 @@ user_send_packet(Acc, From, To, #xmlel{name = <<"message">>} = Msg) ->
 user_send_packet(Acc, _From, _To, _Packet) ->
     Acc.
 
+-spec inbox_unread_count(Acc :: mongooseim_acc:t(), To :: jid:jid()) -> mongooseim_acc:t().
+inbox_unread_count(Acc, To) ->
+    Res = (catch mongoose_acc:get(inbox, unread_count, Acc)),
```

Comment:
Could we use:
```suggestion
    Res = mongoose_acc:get(inbox, unread_count, undefined, Acc),
```
here and handle the `undefined` value later in https://github.com/esl/MongooseIM/pull/2078/files#diff-65d7377e05f0fb0e94fb246102db48bbR322 instead of the `EXIT...` ?